.github/workflows/triage-bot.yaml

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+name: 'Radius Triage Bot'
+on:
+  issues:
+    types: [labeled, unlabeled]
+permissions:
+  contents: read
+  issues: write
+jobs:
+  build:
+    name: 'Process Triage Labels'
+    runs-on: ubuntu-latest
+    env: 
+      RADIUS_TRIAGE_BOT_APP_ID: 417813
+    steps:
+    - uses: tibdex/github-app-token@v1
+      id: get_installation_token
+      with: 
+        app_id: ${{ env.RADIUS_TRIAGE_BOT_APP_ID }}
+        private_key: ${{ secrets.RADIUS_TRIAGE_BOT_PRIVATE_KEY }}
```

Comment:
Something new here is using a GitHub **app** instead of a PAT. We don't actually require a PAT for this use-case. However, using the app is still handy because it lets us control the experience in some pretty cool ways.